.github/workflows/changed-packages.yml

```diff
@@ -28,6 +28,14 @@ jobs:
         run: |
           git fetch origin main --depth 1 &&
           git merge-base --is-ancestor origin/main @;
+      - name: Install dependencies
```

Comment:
This should populate the cache early and prevent the test for running saving us some time on if the build fails.